### PR TITLE
rename e2e-azure to e2e-azure-sdn

### PR DESCRIPTION
**Pull request by jluhrsen**
Signed-off-by: Jamo Luhrsen <jluhrsen@gmail.com>

### CODE REVIEW

**Comment by jluhrsen** [comment]
I think this is  ready now

File: core-services/release-controller/_releases/priv/release-ocp-4.12.json [hunk 1]
```diff
@@ -125,25 +125,25 @@
...
+                "name": "periodic-ci-openshift-release-master-nightly-4.12-e2e-azure-csi-priv"
             }
         },
-        "azure-csi": {
+        "azure-ovn": {
             "disabled": true,
             "optional": true,
             "prowJob": {
-                "name": "periodic-ci-openshift-release-master-nightly-4.12-e2e-azure-csi-priv"
+                "name": "periodic-ci-openshift-release-master-ci-4.12-e2e-azure-ovn-priv"
```
Comment (wking) [inline core-services/release-controller/_releases/priv/release-ocp-4.12.json:139]: why do we have a `...-ci-...` job in this nightly blocker/informer set?

Comment (jluhrsen) [reply core-services/release-controller/_releases/priv/release-ocp-4.12.json:139]: not sure. let me see what's up?

Comment (jluhrsen) [reply core-services/release-controller/_releases/priv/release-ocp-4.12.json:139]: this is leftover from a mistake in this [commit](https://github.com/openshift/release/commit/087ae86ff0d5fe38c99af222182ab890d5a61b59).
I'll address it with it's own PR.

Comment (wking) [reply core-services/release-controller/_releases/priv/release-ocp-4.12.json:139]: it was that way before, so :shrug: no need to hold this PR.

/approve

**Pull request merged**